### PR TITLE
NV14 integration - DO NOT MERGE

### PR DESCRIPTION
**Pull request by kilrah**
Merge being done with the main goal of not breaking existing stuff.
Trying to keep the improvements that were made by @bsongis early in the FlySky repo

- GUI made independent at least for now
- Bitmaps duplicated since cmake currently selects folders automagically
- ADC driver and internal module driver moved out of common for this platform due to unknown status and mergeability

### CODE REVIEW

File: radio/src/CMakeLists.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 include(CMakeForceCompiler)
 include(Bitmaps)
 
-set(PCB_TYPES X7 XLITE X9D X9D+ X9E X10 X12S SKY9X 9XRPRO AR9X)
+set(PCB_TYPES X7 XLITE X9D X9D+ X9E X10 X12S SKY9X 9XRPRO NV14)
```
Comment (schwabe) [inline radio/src/CMakeLists.txt]: Is there a good reason to remove AR9X?

Comment (kilrah) [reply radio/src/CMakeLists.txt]: Nope, merge error, thanks

**Comment by Ask1me** [comment]
Hi.. apologize for my noob Q : i received this radio last week, i want to use it with multiprotocol modules, Does the OpenTx firmware for nv14 support multiprotocol modules such as Jumper JP4in1 or Irx4?

**Comment by 3djc** [comment]
This is clearly not the place for such questions. You did ask this very same question at the right place in your RCGroups post yesterday, so either wait patiently a response threre, or take some time to read more post in that thread to find the answer by yourself

**Comment by Ask1me** [comment]
Yes I know sir and I'm sorry, I have to ask the same question here, I have read all the post in rcgroups one by one and I am confused anyone said that it does not support multiprotocol modules and some say it can, I hope the github forum can give me definite answer, please? 

PS: if my question does not please just delete it thanks

**Comment by 3djc** [comment]
We certainly cannot answer that, since we are not yet distributing nv14 firmware. What is currently available is early beta software comming from the vendor themselves. You should check with them what they have included or not.

**Comment by evildronepilot** [comment]
what is the progress on the NV14 Integration.... when can we hope to get it please?

**Comment by backman8571** [comment]
> what is the progress on the NV14 Integration.... when can we hope to get it please?

Same here when will we see NV14

**Comment by evildronepilot** [comment]
Hold on 1 sec... the support  originally been assigned to 2.3 has something changed? ??

**Comment by lpb06** [comment]
any info why has this case has benn closed?

**Comment by 3djc** [comment]
Because is not ready at all, it will reopened when something 'workable' is ready. It does not means we are less commited to do NV14

**Comment by kilrah** [comment]
This PR was just to track work done, as labeled. Since so much has changed by now it makes no sense leaving it open. It means nothing to actual NV14 support.

**Pull request closed without merging**